src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/BreadcrumbBuilder.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import {Breadcrumb} from 'sulu-admin-bundle/components';
+import {translate} from 'sulu-admin-bundle/services';
+import type {BreadcrumbItems} from './types';
+
+type Props = {
+    breadcrumb: ?BreadcrumbItems,
+    onNavigate: (collectionId?: string | number) => void,
+};
+
+export default class BreadcrumbBuilder extends React.PureComponent<Props> {
```

Comment:
`BreadcrumbBuilder` as a component name sounds a bit strange to me... A builder is something more like a factory class. If we can't come up with a better name we started to prefix them, so it would be `MediaBreadcrumb`

Reply:
Or better said `CollectionBreadcrumb`, since it is about collections and not media 🙈